transformer_engine/common/gemm/cublaslt_gemm.cu

```diff
@@ -182,6 +182,10 @@ void cublas_gemm(const Tensor *inputA,
     NVTE_CHECK_CUBLAS(cublasLtMatmulDescSetAttribute(operationDesc,
                                                      CUBLASLT_MATMUL_DESC_EPILOGUE_AUX_LD,
                                                      &ld_gelumat, sizeof(ld_gelumat)));
+    const cudaDataType_t aux_type = get_cuda_dtype(outputPreGelu->data.dtype);
+    NVTE_CHECK_CUBLAS(cublasLtMatmulDescSetAttribute(operationDesc,
+                                                     CUBLASLT_MATMUL_DESC_EPILOGUE_AUX_DATA_TYPE,
+                                                     &aux_type, sizeof(aux_type)));
```

Comment:
Why didn't we need that before?

Reply:
If we don't set aux_type, it is assumed to be same as output type. But when output is FP8, aux can be FP16 or FP8.